### PR TITLE
Adding some blogs and youtube videos from web to readme

### PR DESCRIPTION
**Pull request by LalatenduMohanty**
Signed-off-by: Lalatendu Mohanty lmohanty@redhat.com

### CODE REVIEW

**Comment by LalatenduMohanty** [comment]
@bexelbie @praveenkumar @navidshaikh ping

File: README.rst [hunk 1]
```diff
@@ -211,6 +212,14 @@ We welcome new ideas, suggestions, issues and pull requests.  Want to be more in
 Documentation is written using `reStructuredText`_. An `online
 reStructuredText editor`_ is available.
 
+On the web:
+==========
+
+* Using OpenShift in ADB : http://www.projectatomic.io/blog/2016/05/App-Development-on-OpenShift-using-ADB
```
Comment (bexelbie) [inline README.rst]: Using OpenShift in the ADB:

File: README.rst [hunk 2]
```diff
@@ -211,6 +212,14 @@ We welcome new ideas, suggestions, issues and pull requests.  Want to be more in
...
+* Using Kubernetes in ADB: http://www.projectatomic.io/blog/2016/04/k8s-adb-usage/
```
Comment (bexelbie) [inline README.rst]: Using Kubernetes in the ADB

File: README.rst [hunk 3]
```diff
@@ -211,6 +212,14 @@ We welcome new ideas, suggestions, issues and pull requests.  Want to be more in
...
+* Introduction to ADB from DevConf.cz 2016: https://www.youtube.com/watch?v=jxFw6qnGaRk
```
Comment (bexelbie) [inline README.rst]: Introduction to the ADB from DevConf.cz 2016 (video):

File: README.rst [hunk 4]
```diff
@@ -211,6 +212,14 @@ We welcome new ideas, suggestions, issues and pull requests.  Want to be more in
...
+* OpenShift Quickstart: https://www.youtube.com/watch?v=H58prwM3IbE
```
Comment (bexelbie) [inline README.rst]: Openshift in the ADB Quickstart (video):

**Comment by LalatenduMohanty** [comment]
@bexelbie updated the PR as per review comments.

**Comment by navidshaikh** [comment]
LGTM!

**Comment by bexelbie** [comment]
LGTM

**Comment by bexelbie** [comment]
manually merged - didn't close automatically

**Pull request closed without merging**